openpype/hosts/houdini/plugins/publish/validate_review_colorspace.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.houdini.api.action import SelectROPAction
+from openpype.hosts.houdini.api.colorspace import get_default_display_view_colorspace  # noqa
+
+import os
+import hou
+
+
+class SetDefaultViewSpaceAction(RepairAction):
+    label = "Set default view space"
+    icon = "mdi.monitor"
+
+
+class ValidateReviewColorspace(pyblish.api.InstancePlugin,
+                               OptionalPyblishPluginMixin):
+    """Validate Review Colorspace parameters.
+
+    It checks if 'OCIO Colorspace' parameter was set to valid value.
+    """
+
+    order = pyblish.api.ValidatorOrder + 0.1
+    families = ["review"]
+    hosts = ["houdini"]
+    label = "Validate Review Colorspace"
+    actions = [SetDefaultViewSpaceAction, SelectROPAction]
+
+    optional = True
+
+    def process(self, instance):
```

Comment:
I'm pretty sure this is missing:
```suggestion
    def process(self, instance):
        if not self.is_active(instance.data):
            return
```

Reply:
Thanks for telling me! 
I thought IT was automated.